modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/DDMStructureLocalServiceTest.java

```diff
@@ -297,6 +356,21 @@ public void testDeleteStructure() throws Exception {
 				structure.getStructureId()));
 	}
 
+	@Test
+	public void testDeleteStructureByExternalReferenceCode() throws Exception {
+		DDMStructure ddmStructure = _addStructure(
+			RandomTestUtil.randomString());
+
+		_ddmStructureLocalService.deleteStructureByExternalReferenceCode(
+			ddmStructure.getExternalReferenceCode(), ddmStructure.getGroupId(),
+			ddmStructure.getClassNameId());
+
+		Assert.assertNull(
+			_ddmStructureLocalService.fetchStructureByExternalReferenceCode(
+				ddmStructure.getExternalReferenceCode(),
+				ddmStructure.getGroupId(), ddmStructure.getClassNameId()));
+	}
+
```

Comment:
We will remove this test because we will remove the method deleteStructureByExternalReferenceCode() from the class.

Reply:
Sure, already removed!